pkgdown/_pkgdown.yml

```diff
@@ -29,15 +31,25 @@ template:
 url: https://appsilon.github.io/shiny.telemetry/
 
 navbar:
+  type: light
   bg: primary
+  fg: "#262323"
   structure:
-    left: [home, reference, changelog]
+    left: [tutorials, reference, changelog]
     right: [search, github, twitter, mastodon]
   components:
-    home:
-      icon: fa-home
-      text: "Start"
-      href: index.html
+    tutorials:
```

Comment:
Please change to `how-to-guides`, as in the future we will probably also have some tutorials here. For reference: https://docs.divio.com/documentation-system/

Reply:
Done in 0283f27